lib/codecs/src/decoding/mod.rs

```diff
@@ -361,6 +378,14 @@ impl DeserializerConfig {
         }
     }
 
+    /// Returns an appropriate default framing config for the given deserializer with message based inputs.
+    pub fn default_message_based_framing(&self) -> FramingConfig {
+        match self {
+            DeserializerConfig::Gelf(_) => FramingConfig::ChunkedGelf(Default::default()),
```

Comment:
Where should I document that when setting `decoding.codec=gelf`, `framing.method` defaults to `chunked_gelf`?

Reply:
Unfortunately I don't think there is a good place to document this at the moment 😞 I think the codec system is complicated enough it'd make sense to have a dedicated page in the docs for it, but that's out-of-scope of this change.